cypress/e2e/fee-fine/waive-button-behavior-fee-fine-details-page.cy.js

```diff
@@ -0,0 +1,116 @@
+import uuid from 'uuid';
+import moment from 'moment/moment';
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UsersOwners from '../../support/fragments/settings/users/usersOwners';
+import ManualCharges from '../../support/fragments/settings/users/manualCharges';
+import PaymentMethods from '../../support/fragments/settings/users/paymentMethods';
+import UserEdit from '../../support/fragments/users/userEdit';
+import NewFeeFine from '../../support/fragments/users/newFeeFine';
+import UsersSearchPane from '../../support/fragments/users/usersSearchPane';
+import UsersCard from '../../support/fragments/users/usersCard';
+import UserAllFeesFines from '../../support/fragments/users/userAllFeesFines';
+import WaiveReasons from '../../support/fragments/settings/users/waiveReasons';
+import WaiveFeeFineModal from '../../support/fragments/users/waiveFeeFineModal';
+
+describe('Waive Fees/Fines', () => {
+  const testData = {
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+    ownerData: {},
+  };
+  const feeFineType = {};
+  const paymentMethod = {};
+  let userData;
+  let feeFineAccount;
+  const waiveReason = WaiveReasons.getDefaultNewWaiveReason(uuid());
+
+  before('Create test data', () => {
+    cy.getAdminToken();
+    ServicePoints.createViaApi(testData.servicePoint);
+    UsersOwners.createViaApi(UsersOwners.getDefaultNewOwner())
+      .then(({ id, owner }) => {
+        testData.ownerData.name = owner;
+        testData.ownerData.id = id;
+      })
+      .then(() => {
+        UsersOwners.addServicePointsViaApi(testData.ownerData, [testData.servicePoint]);
+        ManualCharges.createViaApi({
+          ...ManualCharges.defaultFeeFineType,
+          ownerId: testData.ownerData.id,
+        }).then((manualCharge) => {
+          feeFineType.id = manualCharge.id;
+          feeFineType.name = manualCharge.feeFineType;
+          feeFineType.amount = manualCharge.amount;
+        });
+        PaymentMethods.createViaApi(testData.ownerData.id).then(({ name, id }) => {
```

Comment:
Why do you need this?

Reply:
Before task update the requirement was like that. Now I will fix the both, the prevoius tasks are the same. C462-C463

Reply:
Before task update the requirement was like that. Now I`ll fix it, the previous tasks are the same c462-c463